Basic/DynamicAddressablesNetworkPrefabs/README.md

```diff
@@ -1,27 +1,82 @@
+# Sample Overview
 
-There are several limitations to this API:
-- If you have NetworkConfig.ForceSamePrefabs enabled, you can only modify your prefab lists before starting
+In this sample, learn more about:
+- The dynamic prefabs system
+- Loading prefabs with addressables
+<br><br>
+
+## Exploring the Sample
+
+Each scene in the project showcases a different, isolated feature of the API, allowing for easy extraction into other projects. We suggest exploring them in order to get a good understanding of the flow of dynamically loading and spawning network prefabs. The use-cases available in this sample are based around the current known limitations.
+<br><br>
+
+### Each Scene:
+
+- Scene 00_Preloading Dynamic Prefabs
+  - This is the simplest case of a dynamic prefab - we instruct all game instances to load a network prefab (it can be just one, it could also be a set of network prefabs) and inject them to NetworkManager's NetworkPrefabs list before starting the server.
+  <br><br>
+
+- Scene 01_Connection Approval Required For Late Joining
+  - An optional use-case scenario that walks through what a server would need to validate from a client when dynamically loading network prefabs. Other use-cases don't allow for reconciliation after the server has loaded a prefab dynamically, whereas this one enables this functionality. 
+  <br><br>
+
+- Scene 02_Server Authoritative Preload All Prefabs Asynchronously
+  - A simple use-case where the server notifies all clients to preload a collection of network prefabs. The server will not invoke a spawn in this use-case, and will incrementally load each dynamic prefab, one prefab at a time.
+  <br><br>
+
+- Scene 03_Server Authoritative Try Spawn Synchronously
+  - A dynamic prefab loading use-case where the server instructs all clients to load a single network prefab, and will only invoke a spawn once all clients have successfully completed their respective loads of said prefab. The server will initially send a ClientRpc to all clients, begin loading the prefab on the server, will await acknowledgement of a load via ServerRpcs from each client, and will only spawn the prefab over the network once it has received an acknowledgement from every client, within `m_SynchronousSpawnTimeoutTimer` seconds.
+  <br><br>
+
+- Scene 04_Server Authoritative Spawn Dynamic Prefab Using Network Visibility
+  - A dynamic prefab loading use-case where the server instructs all clients to load a single network prefab via a ClientRpc, will spawn said prefab as soon as it is loaded on the server, and will mark it as network-visible only to clients that have already loaded that same prefab. As soon as a client loads the prefab locally, it sends an acknowledgement ServerRpc, and the server will mark that spawned NetworkObject as network-visible to that client.
```

Comment:
```suggestion
  - A dynamic prefab loading use-case where the server instructs all clients to load a single network prefab via a ClientRpc, will spawn said prefab as soon as it is loaded on the server, and will mark it as network-visible only to clients that have already loaded that same prefab. As soon as a client loads the prefab locally, it sends an acknowledgement ServerRpc, and the server will mark that spawned NetworkObject as network-visible to that client.
  - This makes sure that no client can block the spawn of certain objects by making it visible as soon as specific clients have loaded that prefab. This is great for network reactivity, but might create inconsistent world views for each client depending on whether you've loaded that prefab or not. Using that technique for boss loading could create a situation where your player sees its health decreasing rapidly for no visible reason because the boss is loaded server side and hitting you, but you haven't loaded it yet and it's invisible on your client.
```